test/e2e/common.go

```diff
@@ -39,6 +40,24 @@ func LogFromFile(logFile string) {
 	Logf(string(data))
 }
 
+func logTable(title string, columnNames []string, rows [][]string) {
+	getRowFormatted := func(row []string) string {
+		rowFormatted := row[0]
+		for i := 1; i < len(row); i++ {
+			rowFormatted = fmt.Sprintf("%s\t%s", rowFormatted, row[i])
+		}
```

Comment:
From the [docs](https://pkg.go.dev/text/tabwriter#Writer) it looks like it is important that each cell ends with a tab.

```suggestion
		rowFormatted := ""
		for i, _ := range row {
			rowFormatted = fmt.Sprintf("%s\t%s\t", rowFormatted, row[i])
		}
```